transit_odp/transmodel/migrations/0006_service_service_type.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.20 on 2023-11-15 10:33
```

Comment:
Can we please merge the two migration scripts in one for the service type model changes?